indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -637,6 +646,25 @@
    ;; associated collections stored as EDN gzipped and base64 encoded for retrieving purpose
    :collections-gzip-b64 (m/not-indexed (m/stored m/string-field-mapping))})
 
+(defmapping service-mapping :service
+  "Defines the elasticsearch mapping for storing services."
+  {:_id  {:path "concept-id"}}
+  {:concept-id (-> m/string-field-mapping m/stored m/doc-values)
+   :revision-id (-> m/int-field-mapping m/stored m/doc-values)
+   ;; This is used explicitly for sorting. The values take up less space in the
```

Comment:
This comment is irrelevant now. By using doc-values we are not using the fielddata cache. This concept-seq-id was used when we were trying to reduce the size of the fielddata cache any way we could.

We should not use it for services as it's just increasing the size of the index and adding complexity.

Reply:
Removed `concept-seq-id` field from mapping.